### PR TITLE
Fill input with state data

### PR DESCRIPTION
**Pull request by renancarvalho**
## Proposed Changes

Read value from store on setup step and fill the inputs, it will help when navigating back/forward the steps

## Test Instructions

- Pull this branch and run `yarn start` from the root folder
- Navigate to `setup/intro?flow=link-in-bio or setup/intro?flow=newsletter`
- Advance further on the flow until the domain step
- Navigate back using the browser back button
- Check if the inputs are filled with data

This PR needs to be merged before or together with [this](https://github.com/Automattic/wp-calypso/pull/67252)

### CODE REVIEW

**Review by heavyweight — APPROVED** [review]
Tested it for the flow and looks good!!

File: client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx [hunk 1]
```diff
@@ -6,13 +6,13 @@ import { createInterpolateElement } from '@wordpress/element';
 import { Icon } from '@wordpress/icons';
 import { useI18n } from '@wordpress/react-i18n';
 import React, { FormEvent, useEffect } from 'react';
+import { base64ImageToBlob } from 'calypso/../packages/onboarding/src/setup-tailored-site-after-creation';
```
Comment (alshakero) [inline client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx]: This should be 
```suggestion
import { base64ImageToBlob } from '@automattic/onboarding';
```

And if it's not exported in the package, it should be.

**Review by alshakero — CHANGES_REQUESTED** [review]

File: client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx (hunk 1 above)
Comment (renancarvalho) [reply client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx]: The next commit will address it =) thank you!
🎖️

**Review by alshakero — APPROVED** [review]
Works perfectly.

**Pull request merged**